.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+Tag name (required for main):
```

Comment:
I would change this to say `release branches`:

```suggestion
Tag name (required for release branches):
```

Reply:
changed!